pages/_app.js

```diff
@@ -1,10 +1,29 @@
 import GlobalStyle from "../styles";
+import { SWRConfig } from "swr";
+
+const fetcher = async (url) => {
+  const response = await fetch(url);
+  if (!response.ok) {
+    const error = new Error("An error occurred while fetching the data.");
+    // Attach extra info to the error object.
+    error.info = await res.json();
+    error.status = res.status;
+    console.log(error);
```

Comment:
```suggestion
    error.info = await response.json();
    error.status = response.status;
```
You can remove the console.log.